azuki-core/src/main/kotlin/com/anaplan/engineering/azuki/core/runner/ImplementationInstance.kt

```diff
@@ -67,17 +96,12 @@ interface ImplementationInstance<
                     createInstancesFromConfiguredJars()
                 } else {
                     Log.trace("Loading implementation instances from classpath")
+                    if (havePersistenceVerificationInstance) {
+                        Log.warn("It not advised to mix classpath loading with the user of persistence verification instance. There may be unexpected classloading issues!")
```

Comment:
⛏️ Looks like there might be a typo here.
```suggestion
                        Log.warn("It not advised to mix classpath loading with the use of persistence verification instance. There may be unexpected classloading issues!")
```